molecule/default/python.yml

```diff
@@ -1,11 +1,15 @@
 ---
 - hosts: all
-  name: Install python2/python3 and pip2/pip3
+  name: Install python2 and pip2
```

Comment:
Why did you change this?  With the given `vars` the roles included below will in fact install both versions.

Reply:
Also, if Python is a real dependency of this role then it should be listed as such in `meta/main.yml` as well as in `README.md`, and you should let the installation take place that way instead of via the `prepare.yml` playbook.

Reply:
@jsf9k I think I did things correctly in https://github.com/cisagov/ansible-role-mongo/pull/12/commits/c4d66bed20078a745cb286ac282a57abf25a1cc9, but please double-check to make sure.

Reply:
I think this is correct now.